cobc/error.c

```diff
@@ -603,21 +634,18 @@ configuration_error (const char *fname, const int line,
 }
 
 /* Generic warning/error routines */
-enum cb_warn_val
-cb_warning_x (const enum cb_warn_opt opt, cb_tree x, const char *fmt, ...)
+static enum cb_warn_val
+cb_warning_x_ (const enum cb_warn_opt opt, cb_tree x, const char *fmt, va_list ap)
 {
-	va_list ap;
 	const enum cb_warn_val pref = cb_warn_opt_val[opt];
```

Comment:
It seems that this is duplicate, I suggest to have that code and the "disabled check" only here, not in the caller.